pkg/conduit/runtime.go

```diff
@@ -226,17 +213,29 @@ func newServices(
 		standaloneReg,
 	)
 
+	var schemaService schemaregistry.Service
+	switch cfg.Schema.Type {
+	case SchemaTypeConfluent:
+		schemaService = schemaregistry.NewConfluentService(
+			context.Background(), logger, cfg.Schema.Confluent.ConnectionString, cfg.Schema.Confluent.HealthCheckPath,
```

Comment:
The `context.Background()` caught my attention, I noticed we only need to [log the error](https://github.com/ConduitIO/conduit/blob/3484ba00c3a8547cad4ad0a7a5290bf7981f0011/pkg/schemaregistry/confluent.go#L40-L42) which we then ignore 😱 Why don't we return the error and handle it here?

Reply:
Not sure why we don't return an error, I'll make that change. :+1: